<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefernces</class>
 <widget class="QMainWindow" name="Prefernces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:  #808080
}
QTabWidget{
	background-color:  #808080
}
QTabWidget{ border: 0}

QFrame{
	background-color:   #808080
}

QFrame { border: 0; }

QPushButton{
	background-color: #aab7b8
}
QLineEdit{
	background-color: #aab7b8
}
QComboBox{
	background-color: #aab7b8
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Default save location</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="defaultLocationBar">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="saveDefaultLocationBtn">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>10</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browse">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Dsiable &quot;saved image&quot; pop-up</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="savedImgPopUp">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>80</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>APIs</string>
       </attribute>
       <widget class="QPushButton" name="Save">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AuthenticateBtn">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Authenticate</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="wahheavenApiBar">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>321</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Wallheaven API :</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
